dev-python/pyroute2/pyroute2-0.7.6.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} pypy3 )
+PYTHON_REQ_USE="sqlite"
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="A pure Python netlink and Linux network configuration library"
+HOMEPAGE="
+	https://github.com/svinota/pyroute2
+	https://pypi.org/project/pyroute2/
+"
+
+LICENSE="Apache-2.0"
```

Comment:
The package says:

```
license = GPL-2.0-or-later OR Apache-2.0
```

Any reason why it's being limited to Apache-2.0 here?

Reply:
do not remember or know, will move to `LICENSE="|| ( GPL-2+ Apache-2.0 )"`